resources/views/lists/show.blade.php

```diff
@@ -0,0 +1,74 @@
+@extends('layouts/app')
+@section('content')
+
+<main>
+    <div class="container text-center">
+
+
+        <div class="card my-4">
+            <div class="card-body text-left scroll-card">
+                <h4 class="card-title">Some List<h4>
+                        <h6 class="card-subtitle mb-2">May 2020, Hamburg</h6>
+                        <p class="card-text"> Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque aliquam suscipit nobis odio error saepe, et ipsam labore, delectus asperiores quas sint quidem deserunt placeat consequatur quos, voluptate illum architecto?
+                        </p>
+            </div>
+            <div class="card-footer p-0 justify-content-around">
+                <span class="card-icon">&#9749;</span>
+                <span class="card-icon">&#9748;</span>
+                <span class="card-icon">&#9784;</span>
+                <span class="card-icon">&#9785;</span>
+            </div>
+        </div>
```

Comment:
😄 Ok, never mind...

Reply:
There are different types of cards in my app...but I have already started to refactor my html and css. At the moment it still looks like Frankenstein's code